tests/Constraints/MinMaxPropertiesTest.php

```diff
@@ -82,8 +82,26 @@ public function getInvalidTests()
                 }'
             ),
             array(
+                '{}',
+                '{
+                  "type": "object",
+                  "properties": {
+                    "propertyOne": {
+                      "type": "string"
+                    },
+                    "propertyTwo": {
+                      "type": "string"
+                    }
+                  },
+                  "minProperties": 1
+                }'
+            ),
+            array(
                 '{
-                  "value": 1
```

Comment:
both tests might be simplified by not having the `value` wrapping but we chose not to change that to keep the diff simpler